src/styles/Document.js

```diff
@@ -54,14 +54,14 @@ export default css`
 .renderer {
   padding: 8px 0;
   display: flex;
-  max-height: 500px;
+  max-height: var(--api-resource-example-document-max-height, 500px);
   -webkit-transition: all 0.4s 0.1s ease-in-out;
   -moz-transition: all 0.4s 0.1s ease-in-out;
   -o-transition: all 0.4s 0.1s ease-in-out;
   transition: all 0.4s 0.1s ease-in-out;
 }
 
-.collapse {
+.renderer.collapse {
```

Comment:
Are we 100% sure this doesn't cause any problems or break anything, by making it a more specific selector?

Reply:
Not cause problems, but I deleted for avoid noise